src/ThreeDSAuth.res

```diff
@@ -5,8 +5,22 @@ let make = () => {
   let (openModal, setOpenModal) = React.useState(_ => false)
   let (loader, setloader) = React.useState(_ => true)
 
+  let threeDsAuthoriseUrl = React.useRef("")
+  let (expiryTime, setExpiryTime) = React.useState(_ => 60000.0)
```

Comment:
does this have to be a float? I see that the decrements are in multiples of 1000.0 anyway
This could simply be an int

Reply:
i thing this apporach itself is not needed .. will be adding a new approach with just settimout